spec/features/user_following_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+feature 'User following' do
+  scenario 'user follows another user' do
+    user = Fabricate(:user)
+    another_user = Fabricate(:user)
+    video = Fabricate(:video)
+    review = Fabricate(:review, video: video, user: another_user)
+    
+    sign_in(user)
+    visit video_path(video)
+    click_link another_user.full_name
+    click_link 'Follow'
+    expect(page).to have_css('button', text: 'Following')
+    click_link 'People'
+    expect(page).to have_link(another_user.full_name)
+    
+    unfollow(user, another_user)
+    expect(page).not_to have_link(another_user.full_name)
```

Comment:
The inverse of `to have_link` is actually `to have_no_link`. http://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Matchers#has_no_link%3F-instance_method